driver/driver.hpp

```diff
@@ -150,11 +150,7 @@ inline void PadBufferSize(size_t& sz, int datatype_sz)
     printf("Supported Base Arguments: conv[fp16|int8|bfp16|fp8|bfp8], CBAInfer[fp16], "
            "pool[fp16], lrn[fp16], "
            "activ[fp16], softmax[fp16], bnorm[fp16], rnn[fp16], gemm, ctc, dropout[fp16], "
-           "tensorop[fp16], reduce[fp16,fp64]"
-#ifdef MIOPEN_BETA_API
-           ", layernorm[bf16, fp16, fp32]"
-#endif
```

Comment:
Please see my previous comment and keep `", layernorm[bf16, fp16, fp32]"`, just remove the conditional.

Reply:
In this case layernorm[bfp16, fp16] still remains and only the condition has been deleted. Please explain if your intention is different.

Reply:
@seungmanhan Oops, wrong review comment [resolved]